app/lib/report_helper.rb

```diff
@@ -970,7 +1040,7 @@ def taxonomy_details(pipeline_run_id, background_id, params)
     rows_passing_filters = rows.length
 
     # Compute sort key and sort.
-    sort_by = decode_sort_by(params[:sort_by]) || decode_sort_by(DEFAULT_SORT_PARAM)
+    sort_by = decode_sort_by(sort_by) || decode_sort_by(DEFAULT_SORT_PARAM)
```

Comment:
where is sort_by coming from without the `params[:sort_by]`?

Reply:
It is hard coded to the default here: 

```    
report_info = ReportHelper.external_report_info(
      pipeline_run_id,
      background_id,
      TaxonScoringModel::DEFAULT_MODEL_NAME,
      ReportHelper::DEFAULT_SORT_PARAM
    )
```